cmd/kubeadm/app/util/system/kernel_validator.go

```diff
@@ -78,7 +78,7 @@ func (k *KernelValidator) Validate(spec SysSpec) (error, error) {
 
 // validateKernelVersion validates the kernel version.
 func (k *KernelValidator) validateKernelVersion(kSpec KernelSpec) error {
-	glog.V(1).Info("Validating kernel version")
+	fmt.Printf("[%s] Validating kernel version\n", k.Name())
```

Comment:
my be better to remove this call and the other one [1].

and then also convert this line to use fmt.Printf()
https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/util/system/validators.go#L44